test/infrastructure/inmemory/internal/server/api/handler.go

```diff
@@ -535,6 +543,9 @@ func getAPIResourceList(req *restful.Request) *metav1.APIResourceList {
 		if req.PathParameter("group") == "rbac.authorization.k8s.io" && req.PathParameter("version") == "v1" {
 			return rbacv1APIResourceList
 		}
+		if req.PathParameter("group") == "apps" && req.PathParameter("version") == "v1" {
+			return appsV1ResourceList
```

Comment:
Could probably do some sort of mapping for groupVersion => resourceList here. Not sure if it's worth doing now or how many more paths we'll have to add in these functions.

Reply:
I think it is ok for now, let's reconsider if this grows more